docs/aws/services/ebs.md

```diff
@@ -0,0 +1,23 @@
+# Amazon Elastic Block Store (Amazon EBS)
+
+## Service Details
+
+Amazon Elastic Block Store (Amazon EBS) provides block level storage volumes for use with EC2 instances.
+
+## Assessment Notes
+* Encryption at rest should be enabled
+  * Default encryption should be enabled for each region at the account level. This can only be checked through the console - [https://aws.amazon.com/blogs/aws/new-opt-in-to-default-encryption-for-new-ebs-volumes/](https://aws.amazon.com/blogs/aws/new-opt-in-to-default-encryption-for-new-ebs-volumes/)
```

Comment:
We're saying that it can only be checked through the console, but below we have a command for it?